version.md

```diff
@@ -2,70 +2,120 @@
+* 422 Unprocessable Entity - if there are already an inprogress submission representing a new version in the same version history of the item 
 
 ## Get single version
 
 **GET /api/versioning/versions/<:versionId>**
 
-Provide version information for the version id.
+Provide the version information associated to a specific id. The returned attributes depend on the user permissions and system configuration.
+The following example shows all the available attributes as exposed to an administrator
 
 ```json
 {
-  "id": "102",
-  "version": "2",
+  "id": "345",
+  "version": "101",
   "type": "version",
   "created": "2015-11-03T09:44:46.617",
   "summary": "Fixing some typos in the abstract",
   "submitterName": "LastName, FirstName",
   "_links": {
     "versionhistory": {
-      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versionhistories/10"
+      "href": "https://demo7.dspace.org/server/api/versioning/versionhistories/1"
     },
     "self": {
-      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versions/101"
+      "href": "https://demo7.dspace.org/server/api/versioning/versions/345"
     },
     "item": {
-      "href": "https://dspace7.4science.it/dspace-spring-rest/api/versioning/versions/101/item"
+      "href": "https://demo7.dspace.org/server/api/versioning/versions/345/item"
     }
   }
-}
-```         
+}         
+```
 
-The `submitterName` attribute is only exposed if you are logged in as an administrator or if the `versioning.item.history.include.submitter` configuration property is set to true.
+The `submitterName` attribute is only exposed if you are logged in as an administrator of the linked item or if the `versioning.item.history.include.submitter` configuration property is set to true.
 
 Status codes:
 * 200 OK - if the version exists and is accessible by the current user
 * 401 Unauthorized - if you are not authenticated and versioning is not public
 * 403 Forbidden - if you are not logged in with sufficient permissions and versioning is not public
-* 404 Not found - if the version doesn't exist
+* 404 Not found - if the version doesn't exist or was deleted
+
+### Get single version for item
+Please follow the version link in [the item endpoint](items.md#get-single-version-for-item)
 
-## Get single version for item
+## Remove version
 
-**GET /api/core/items/{:item-uuid}/version**
+To delete a version you need to [delete the item](items.md#deleting-an-item) linked to it.
 
-See [the item endpoint](items.md#get-single-version-for-item)
+## Update version (patch operations)
 
-## Remove version
+Only the summary property can be updated by item administrators, all the other properties are **READ-ONLY**. Attempt to update any other information will result in a 422 error.
+
+To update the metadata of the corresponding item the appropriate method in the items, workspaceitems or workflowitems endpoints need to be used according to the item status.
+
+**PATCH **
```

Comment:
Tiny fix, the formatting is broken on this line. I think you meant for this to be `**PATCH**` (i.e. remove extra space)